pkg/controllers/job/state/running.go

```diff
@@ -62,6 +63,7 @@ func (ps *runningState) Execute(action v1alpha1.Action) error {
 			minSuccess := ps.job.Job.Spec.MinSuccess
 			if minSuccess != nil && status.Succeeded >= *minSuccess {
 				status.State.Phase = vcbatch.Completed
+				metrics.RegisterJobCompleted(ps.job.Job.Namespace)
```

Comment:
`metrics.RegisterJobCompleted(fmt.Sprintf("%s/%s", ps.job.Job.Namespace, ps.job.Job.Name))`

Same for other parts